src/sentry/integrations/jira/webhooks/base.py

```diff
@@ -31,11 +34,70 @@ def dispatch(self, request: Request, *args, **kwargs) -> Response:
     def handle_exception(
         self, request: Request, exc: Exception, handler_context: Mapping[str, Any] | None = None
     ) -> Response:
+        handler_context = handler_context or {}
+
         if isinstance(exc, (AtlassianConnectValidationError, JiraTokenError)):
             return self.respond(status=status.HTTP_400_BAD_REQUEST)
-        # Perhaps it makes sense to do this in the base class, however, I'm concerned
-        # it would create too many errors at once and may be grouped together
-        logger.exception("Unclear JIRA exception.")
+
+        # Atlassian has an automated tool which tests to make sure integrations with Jira
+        # (like ours) pass certain security requirements, which leads them to probe certain
+        # of our endpoints to make sure we're not accepting insecure GET requests. Not
+        # actionable on our part and therefore not worth sending to Sentry.
+        if isinstance(
+            exc, MethodNotAllowed
+        ) and "github.com/atlassian-labs/connect-security-req-tester" in request.headers.get(
+            "User-Agent", ""
+        ):
+            logger.info(
+                "Atlassian Connect Security Request Tester tried disallowed method",
+                extra={"path": request.path, "method": request.method},
+            )
+            return self.respond(status=status.HTTP_405_METHOD_NOT_ALLOWED)
+
+        # Handle and clean up known errors, and flag unknown ones by logging an exception
+        if isinstance(exc, ApiError):
+            # Pull off "com.atlassian.jira.issue:sentry.io.jira:sentry-issues-glance:status"
+            # and the like - what we really care about is which Jira API endpoint we were
+            # trying to hit
+            jira_api_endpoint = (
+                exc.path.split("com.atlassian.jira")[0] if exc.path else "[unknown endpoint]"
+            )
+
+            # TODO: Should either of these be tags instead, so they're searchable?
+            handler_context["jira_host"] = exc.host
```

Comment:
This one as a tag is useful. Maybe `jira.host`?

Reply:
Done in https://github.com/getsentry/sentry/commit/a9d45fc05e6c5d255c1d6eb6ce759a3b1997e31e. It requires a rebase onto https://github.com/getsentry/sentry/pull/47777, so I'll hold off merging this until that's through. (That's why there are suddenly a ton of extra commits here. They'll go away once that's merged.)